Add gamebosu and edit install method page to new design (#25)
Rurusetto/rurusetto: content/pages/install/index.md

# download rulesets file

> Get rulesets file from status page by go to [rulesets status page](https://rulesets.info/pages/status) and click on `detail & download` button on desired rulesets.

Reviewer: ```suggestion
Get rulesets file from status page by going to [rulesets status page](https://rulesets.info/pages/status) and then clicking on the `detail & download` button for the desired ruleset.
```

> Get rulesets file from status page by go to [rulesets status page](https://rulesets.info/pages/status) and click on `detail & download` button on desired rulesets.

Reviewer 2: ```suggestion
Ruleset files can be obtained at [rulesets status page](https://rulesets.info/pages/status) and clicking `detail & download` button on the desired ruleset.
```

![status page](img/status-download.png)

> A box with a changelog and select ruleset file detail will appear. After finish read about changelog, then click `Download` button at the right-bottom corner of the box to get a ruleset file.

Reviewer 2: A dialog will appear with a changelog. Click the `Download` button at the bottom-right corner to initiate the download.

> A box with a changelog and select ruleset file detail will appear. After finish read about changelog, then click `Download` button at the right-bottom corner of the box to get a ruleset file.

Reviewer: ```suggestion
A box with the changelog of the selected ruleset file details will appear. After you finish reading about changelog click the `Download` button at the right-bottom corner of the box to get the ruleset file.
```

![sentakki release download](img/sentakki-release-download.png)

> A link to download and chnagelog will auto-update every time a ruleset release a new version so you can grab a new release every time a ruleset have updated and not to worry about out-of-date link.

Reviewer 2: The changelog and download link will automatically update when a ruleset is updated.
